tracer/src/Datadog.Trace/Iast/Range.cs

```diff
@@ -26,6 +29,8 @@ public Range(int start, int length, Source? source = null)
 
     public Source? Source { get; }
 
+    public Mark Mark { get; }
```

Comment:
Same here. A better name could be `SecureMarks`

Reply:
Changed in 6f642d9